pkg/scalers/http_scaler.go

```diff
@@ -0,0 +1,145 @@
+package scalers
```

Comment:
Can we integrate the health endpoint to probe to see if it's still healthy or provide a way next to it.

This could be surfaced in the CRD info that it's not calling a health metric source instead of only in the logs.

Reply:
Not sure I understand, can you explain more?